docs/reference/hardware_features.rst

```diff
@@ -0,0 +1,249 @@
+.. meta::
+  :description: This chapter describes the hardware features of the different hardware architectures.
+  :keywords: AMD, ROCm, HIP, hardware, hardware features, hardware architectures
+
+*******************************************************************************
+Hardware features
+*******************************************************************************
+
+This page gives an overview of the different hardware architectures and the
+features they implement. Hardware features do not imply performance, that
+depends on the specifications found in the :doc:`rocm:reference/gpu-arch-specs`
+page.
+
+  .. list-table::
+      :header-rows: 1
+      :name: hardware-features-table
+
+      *
+        - Hardware feature support
+        - RDNA1
+        - CDNA1
+        - RDNA2
+        - CDNA2
+        - RDNA3
+        - CDNA3
```

Comment:
There was a recent introduction of the CDNA4 architecture. Is this something we should add to the table?

Reply:
As long as the GPUs are not released I would refrain from adding information here